README.md

```diff
@@ -1,3 +1,171 @@
 # kube-service-bindings
 
-A repository with examples utilizing [kube-service-binding](https://www.npmjs.com/package/kube-service-bindings) npm package:
+A repository with examples utilizing [kube-service-bindings](https://www.npmjs.com/package/kube-service-bindings) npm package:
+
+- KAFKA
+  - node-rdkafka
+  - kafkajs
+- POSTGRESQL
+  - [pg](./src/pg/README.md)
+  - [odbc](./src/odbc-psql/README.md)
+- REDIS
+  - redis
+  - ioredis
+- MONGODB
+  - [mongoose](./src/mongoose/README.md)
+  - [mongodb](./src/mongodb/README.md)
+- AMQP
+  - [rhea](./src/rhea/README.md)
+- MYSQL
+  - [mysql2](./src/mysql2/README.md)
+  - [odbc](./src/odbc-mysql/README.md)
+  - [mysql](./src/mysql/README.md)
+
+# Basic instructions
+
```

Comment:
The examples listed need some common configuration along with some additional components needed by the
exmple itself. In terms of common configuration they all need an OpenShift cluster and the service binding operator installed. As an example of additional components, the example showing how to use MYSQL with
kube-service-bindings needs a MYSQL database installed.

This section explains how to install those common and additional components.

Reply:
Yes, exactly.
Definitely I would like to rename the title of this part from "Basic Instructions" to "Common Configuration" if you agree.